src/apigility-ui/service/api.service.js

```diff
@@ -94,11 +94,7 @@
       })
       .catch(function (err) {
         growl.error('Unable to create service', {ttl: -1});
-        switch (err.status) {
-          case 409 :
-            return callback(true, 'The service already exists, please choose a different name');
-        }
-        return callback(true, 'I cannot create the REST service, please enter a valid name');
+        return callback(true, 'Unable to create service (Code: ' + err.status + '): ' + err.data.detail);
```

Comment:
I don't see this as an improvement, other than adding the error status and detail.

Having a separate message for 409 status vs other status is important, as it flags to the user when they have attempted to create a service using a name that already exists, and we definitely want to do that.

I'll merge, but when I do so, I'll revert the removal of the switch statement (in both blocks).